app/femr/ui/views/triage/index.scala.html

```diff
@@ -26,6 +26,10 @@
         @if(viewModel.isSearchError) {
             <p>That patient could not be found.</p>
         }
+        @if(viewModel.isLinkToMedical()){
+
+           <p> <a class="btn btn-danger" href="/medical/edit/@viewModel.getPatient().getId()">Patient has an open encounter!</a></p>
```

Comment:
Try something more like this. You may need some css still. The button might need

`display: inline-block;`

`<p>Patient has an open encounter <a class="btn btn-danger" href="/medical/edit/@viewModel.getPatient().getId()">Go To Medical</a></p>`